test

```diff
@@ -6,14 +6,33 @@ function diff_files () {
     git diff origin/master --name-only
 }
 
-function search () {
+function build_commited () {
+    # Build only commited changed
+    #TODO: mode=sync/async
     git diff origin/master --name-only | xargs python3 stats.py --build --files
 }
 
-cmd="$1"
+function build_all () {
+    opt="$1"
+    mode="$2" # sync/async
+    if [ "$opt" = "--master-only" ] || [ "$opt" = "-m" ]; then
+        git checkout master
+        echo "heheheh"
+        python3 stats.py --build --all
+    elif [ "$opt" = "--master-with-commited"] || [ "$opt" = "-mc" ]; then
+        ehco "heheheheh"
+        python3 stats.py --build --all
+    fi
+}
 
-if [ "$cmd" = "--staged-only" ] || [ "$cmd" = "-so" ]; then
-    search
+cmd="$1"
+mode="$2" # sync/async
+opt="$3"
+if [ "$cmd" = "--commited-only" ] || [ "$cmd" = "-c" ]; then
+    build_commited
+elif [ "$cmd" = "--all" ] || [ "$cmd" = "-a" ]; then
+    echo "heheheh"
```

Comment:
LOL

Reply:
xd you found it I tried to delete them all x)